releases/latest/kernel-slim/Dockerfile.ubi.openjdk21

```diff
@@ -0,0 +1,153 @@
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal AS getRuntime
+
+USER root
+
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_SHA={replace_with_correct_sha}
+ARG LIBERTY_DOWNLOAD_URL=https://repo1.maven.org/maven2/io/openliberty/openliberty-kernel/$LIBERTY_VERSION/openliberty-kernel-$LIBERTY_VERSION.zip
+
+ARG VERBOSE=false
+
+# If there is a local copy of the image use that instead
+COPY resources/ /tmp/
+
+# Install Open Liberty
+RUN microdnf -y install shadow-utils wget unzip openssl \
+    && if [ ! -f /tmp/wlp.zip ]; then wget -q $LIBERTY_DOWNLOAD_URL -U UA-Open-Liberty-Docker -O /tmp/wlp.zip; fi \
+    && echo "$LIBERTY_SHA  /tmp/wlp.zip" > /tmp/wlp.zip.sha1 \
+    && sha1sum -c /tmp/wlp.zip.sha1 \
+    && chmod -R u+x /usr/bin \
+    && unzip -q /tmp/wlp.zip -d /opt/ol \
+    && mkdir -p /licenses \
+    && cp /opt/ol/wlp/LICENSE /licenses/ \
+    && adduser -u 1001 -r -g root -s /usr/sbin/nologin default \
+    && chown -R 1001:0 /opt/ol/wlp \
+    && chmod -R g+rw /opt/ol/wlp
+
+FROM icr.io/appcafe/ibm-semeru-runtimes:open-21-jre-ubi9-minimal
+
+USER root
+
+ARG LIBERTY_VERSION=24.0.0.1
+ARG LIBERTY_BUILD_LABEL=replace_with_gm_driver_label
```

Comment:
It should be {replace_with_gm_driver_label} (with curly brackets)

Reply:
Done